command/debug.cpp

```diff
@@ -41,6 +42,18 @@ bool toggle_allow_client_side_projectiles(std::vector<VulpesArg> input){
     return true;
 }
 
+bool get_network_id_from_obj_id(std::vector<VulpesArg> input){
```

Comment:
I assume these return true because Halo's console requires it? Might be worth it to put a note about that somewhere.

Reply:
No reason. Pretty embarrassingly, this is a leftover from the old concept of command handling. The return value is _never_ used.

As described in #43 I want to write a test suite for this command handling code, as it's big, gross and sadly mostly untested.

I want to work on that and then change how the functions in that file work based on what we find there. That will probably end in the requirement of these functions returning a bool to be removed.